src/SerilogWeb.Classic/SerilogWeb.Classic.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>SerilogWeb.Classic</id>
-    <version>$version$</version>
+    <version>2.0.3-private2</version>
```

Comment:
Actually - `Build.ps1` will do this for you - it's the kick-off point for the build.